docs/teams/competition-preparation.md

```diff
@@ -1,22 +0,0 @@
----
-original:
-  authors: Fiona Gillan
-  url: https://github.com/fgillan/sr-team-coord/wiki/Competition-preparation
----
-## Tickets
-
-Liaise with Competition Coordinator about ticket allocation process for teams.  Once system confirmed inform LTCs and Team Leaders and remind them of process periodically to ensure process is being followed.
-
-## Information about the competition
-
-Liaise with Competition Coordinator about information that they would like relayed to the Team Leaders prior to the competition and send email via MailChimp Team Leaders mailist. To include information such as the following:
-
-1. Teams should bring some laptops as we will not be providing any computers/laptops.
-2. Teams should bring any tools that they require to work on their robot while at the competition as we will not be providing any tools.
-3. All of the kit must be returned at the end of the competition (including the white box it comes in). If a team wishes to keep the kit for a little longer they must email info@studentrobotics.org ASAP to request a loan extension. Any team that has not been granted a loan extension before the start of the competition will have to return all of the kit before they leave on Sunday.
-4. Teams should have a good read of section 2.8 and Appendix B of the rules (Safety regulations). All robots will undergo a safety inspection and will not be allowed to compete until they pass this inspection.
-5. Please do let us know if your team has any special requirements with regards to disability access or other matters that need to be accommodated.
```

Comment:
Much of this information seems still relevant and useful, just to check: have we captured this somewhere else now we're dropping it from here?